gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Improved UI for HomePage

### Description

Changed the UI for Homepage

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
python manage.py runserver
```

1. Create a user and mark as interested/going to events
2.Navigate to the Home Page, you should see that the template is changed now

### Miscellaneous Comments

*   Use backticks to write `code`.
*   Whatever else you feel like saying can go here.

### Checklist
